internal/Picker/tests/Picker-specs.js

```diff
@@ -138,6 +162,51 @@ describe('Picker Specs', () => {
 		expect(actual).toBe(expected);
 	});
 
+	test('should register wheel event when \'joined\' if \'value\' is between \'max\' and \'min\'', () => {
```

Comment:
```suggestion
	test('should register wheel event when \'joined\' and \'value\' is between \'max\' and \'min\'', () => {
```
Could you please fix this line?